classes/image/imagemagick.php

```diff
@@ -158,6 +158,14 @@ protected function _mask($maskimage)
 		$this->exec('convert', $command);
 	}
 
+	protected function _blur($radius, $sigma)
+	{
+		extract(parent::_blur($radius, $sigma));
+
+		$image = '"'.$this->image_temp.'"';
+		$this->exec('convert', $image.' -blur '.$radius.'x'.$sigma.' '.$image);
```

Comment:
Formatting to string is already handled in the queue system, like in others methods: https://github.com/novius-os/fuelphp-core/blob/1.7/master/classes/image/driver.php#L848